README.markdown

```diff
@@ -285,6 +428,20 @@ var diameter: Double {
 }
 ```
 
+### Final
+
+Mark classes `final` when inheritance is not intended. Example:
+
+```swift
+// We don't allow subclassing of an equilateral triangle
+final class EquilateralTriangle : Triangle {
+    let sideLength: Int
+    init(sideLength: Int) {
+        self.sideLength = sideLength
+    }
+}
+```
```

Comment:
And if we haven't mentioned it elsewhere: Make types, methods, and properties `private` whenever possible.